### ISSUE TITLE
[VT-92] missing --version option

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#114]
_David Patrick on 2014-06-18T07:44:54Z says:_ 

 There seems to be no easily discoverable way to determine what version of vit you are using. 

a vit --version, or -v option is what might be expected.

and perhaps a keybinding "V" might bring up a version/ diagnostic output.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-06-18T07:44:54Z
Modified: 2014-06-21T06:23:50Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2014-06-21T06:23:50Z says:_ 

 Fixed at d2c027da. As of 6a0fa019, "vit --version", "vit -version", and "vit -v" are all supported.
